ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/IndexConfigManagementService.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.index.v2
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.daml.ledger.participant.state.v1.Configuration
+import com.digitalasset.ledger.api.domain.ConfigurationEntry
+
+import scala.concurrent.Future
+
+/**
+  * Serves as a backend to implement
+  * [[com.digitalasset.ledger.api.v1.admin.config_management_service.ConfigManagementServiceGrpc]]
+  *
+  */
+trait IndexConfigManagementService {
+
+  /** Looks up the current configuration, if set, and the offset at which
+    * it is current. Offset can be used to subscribe to further configuration changes
+    * using [[configurationEntries]]. */
+  def lookupConfiguration(): Future[Option[(Long, Configuration)]]
+
+  /** Retrieve configuration entries. */
+  def configurationEntries(startExclusive: Option[Long]): Source[ConfigurationEntry, NotUsed]
```

Comment:
I had the option of either making it inclusive and making lookupConfiguration return "the next offset" or making this exclusive. Since we're planning on going towards external offsets which we cannot "+1" it makes more sense to make it exclusive.